src/obfl-specification.html

```diff
@@ -1195,6 +1200,72 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd class="markup note">This attribute is only available in the context of a field element.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd class="markup note">For readability, the examples below have plain text
+      instead of braille.</dd>
+    <dd>
+      <p>These are the commonly available variables for use in the <a
+        href="obfl-evaluation-language.html">OBFL Evaluation Language</a>
+        expression:</p>
+      <dl>
+        <dt>page</dt>
+          <dd>The variable holding the current page number. For instance,
+            <code>This is page &lt;evaluate expression="$page"/&gt;</code>.</dd>
+        <dt>volume</dt>
+          <dd>The variable holding the current volume number. For instance,
+            <code>This is volume &lt;evaluate
+            expression="$volume"/&gt;</code>.</dd>
+        <dt>volumes</dt>
+          <dd>The variable holding the total number of volumes. For instance,
+            <code>This book has &lt;evaluate expression="$volumes"/&gt;
+              volumes</code>.</dd>
+        <dt>sheets-in-document</dt>
+          <dd>The variable holding the number of sheets in this document. For
+            instance, <code>This document has &lt;evaluate
+            expression="$sheets-in-document"/&gt; sheets</code>.</dd>
+        <dt>sheets-in-volume</dt>
+          <dd>The variable holding the number of sheets of the current volume.
+            For instance, <code>This volume has &lt;evaluate
+            expression="$sheets-in-volume"/&gt; sheets</code>.</dd>
+      </dl>
+      <p>These common variables can also be used in all use-when conditions.</p>
+    </dd>
+    <dd>
+      <p class="markup">This variable is available in descendant elements of
+        on-volume-start and on-volume-end:</p>
+      <dl>
+        <dt>started-volume-number</dt>
+          <dd>The volume number of the volume that was started or ended.</dd>
```

Comment:
> Refer to the "on-volume-start" and "on-volume-end" sections.

I do that. The line above has a link to on-volume-start and on-volume-end. This link is generated by the `buildSite` command because the previous line has `class="markup"`.

In the html file that is generated you can click on the `on-volume-start` and `on-volume-end` of line 1235 and it takes you right to those elements. And from there, you can click on `evaluate` which takes you back to this overview of all variables.

Reply:
> or ended? In what case is this the number of the previous volume?

@kalaspuffar In case of `on-volume-start` it is the volume that was started, in case of `on-volume-end` it was the volume that has just ended.

Reply:
Great, is there a way to express that I in 1 week can remember that?

Reply:
I will change "or" to "resp.". Then it becomes:

The volume number of the volume that was started resp. ended.

I hope that this will be ok

Reply:
Aaah OK. That's for the preview, very helpful!